README.md

```diff
@@ -139,10 +139,8 @@ In addition to code blocks, some code blocks should be rendered differently depe
 ### Example
 
 <pre>
```

Comment:
This suggestion means to revert deletion (line 142) though it looks like I haven't changed the code.
```suggestion
<pre>
```
```

Reply:
Commit proposed changes as in `Satendra-SR:Fixed-readme-formatting`